src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/SentenceLength.kt

```diff
@@ -1,8 +1,7 @@
 package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models
 
 data class SentenceLength(
-  val days: Int? = null,
-  val weeks: Int? = null,
-  val months: Int? = null,
-  val years: Int? = null,
+  val duration: Int? = null,
+  val units: String? = null,
+  val terms: List<SentenceTerm> = listOf(SentenceTerm()),
```

Comment:
```suggestion
  val terms: List<SentenceTerm> = emptyList(),
```

If there are no terms, then I don't think it makes sense to return a single object with null for all properties...